test/stream.js

```diff
@@ -16,9 +16,9 @@ const pExec = promisify(exec);
 setFixtureDir();
 
 test('buffer', async t => {
```

Comment:
There are two tests in this file which currently checks whether the `input` option can be a `Buffer`.
The first one actually passes a `string` (which is incorrect) while the second passes a `Buffer`.
Maybe we should test passing an `Uint8Array` in both cases instead?